src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -50,6 +53,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         new Gson().fromJson(request.getParameter(PLACE_FILTERS), HashSet.class);
     HashSet<String> userDifficultyStrings =
         new Gson().fromJson(request.getParameter(DIFF_FILTERS), HashSet.class);
+    String numPlacesString = new Gson().fromJson(request.getParameter(NUM_PLACES), String.class);
+    int numPlaces = 3;
+    if (numPlacesString == "Five") {
```

Comment:
I think .equals() is usually used in Java for String comparison (rather than ==).

Reply:
Oh you're totally right, I forgot!